### ISSUE TITLE
Autoredirected

### ISSUE THREAD

**Issue by uluhonolulu** [gitcoinco/gitcoinco#23]
I first visited the issue browser using a link to https://gitcoin.co/explorer?q=consensys. Now whenever I try to get to the whole list (https://gitcoin.co/explorer), I keep being redirected to the original page. 

Latest Chrome with MetaMask enabled.

**Comment by owocki** [comment]
hi @uluhonolulu if you clear the search bar and press enter, itll send you to the whole list

maybe something we should change is that if `q` parameter is not present, it takes you to the whole list (as opposed to the last thing you serached)?

**Comment by uluhonolulu** [comment]
@owocki  Yes, that would be less confusing, because when I click the Explorer link, I expect it to show me all entries.

Also, I noticed that Firefox doesn't do this (show me the last search results).
